docs/guide/adapters/emberznet.md

```diff
@@ -103,6 +103,25 @@ serial:
 
 :::
 
+::: details Z-WAVE>ME Z-Station (EFR32MG21P)
+
+With external antenna.
+
+Requires Z-Station firmware 2024.04.22 or later.
+
+```yaml
+serial:
+    rtscts: false
```

Comment:
Even though this is the default setting, I added it here because setting it to `true` does not work.